assets/ui/scoreHud.ui

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "ScoreHud",
+    "contents": {
+        "type": "relativeLayout",
+        "contents": [
+            {
+                "type": "UIText",
```

Comment:
I guess this is a good start. For good visuals you might need to add more UI widgets here. Also, think about layout capabilities for different screen sizes, etc...